src/main/java/org/carlmontrobotics/robotcode2023/RobotContainer.java

```diff
@@ -105,6 +114,9 @@ private void configureButtonBindingsManipulator() {
     new POVButton(manipulatorController, Manipulator.intakeConePOV).onTrue(new SetArmWristGoalPreset(GoalPos.INTAKE, () -> false, isFront, arm));
     new POVButton(manipulatorController, Manipulator.substationPickupPOV).onTrue(new SetArmWristGoalPreset(GoalPos.STORED, isCube, isFront, arm));
     new POVButton(manipulatorController, Manipulator.intakeCubePOV).onTrue(new SetArmWristGoalPreset(GoalPos.STORED, () -> true, isFront, arm));
+    new Trigger(getForbFlag).onTrue(new InstantCommand(() -> {manipulatorController.setRumble(RumbleType.kBothRumble,1);}));
+    new Trigger(getForbFlag).onFalse(new InstantCommand(() -> {manipulatorController.setRumble(RumbleType.kBothRumble,0);}));
```

Comment:
Your "getForbFlag" is not letting your code build. BEFORE you push make sure your code BUILDS. I'm not sure whether Codespaces has syntax highlighting but make sure there are no red alerts on your ide.